src/Symfony/Component/HttpKernel/Tests/Profiler/FileProfilerStorageTest.php

```diff
@@ -371,7 +371,7 @@ public static function provideExpiredProfiles()
 
         yield 'One unexpired profile' => [
             <<<CSV
-            token0,127.0.0.0,,http://foo.bar/0,{$oneHourAgo->getTimestamp()},,
+            token0,127.0.0.0,,http://foo.bar/0,{$oneHourAgo->getTimestamp()},,,
```

Comment:
we should not need to change any existing test case (and we might add one that has a virtual type)